.github/workflows/release.yml

```diff
@@ -76,10 +76,10 @@ jobs:
 
       - uses: actions/checkout@v3
 
-      - name: "Set up JDK 20.0.1"
+      - name: "Set up JDK 21.0.4"
```

Comment:
Here and elsewhere: can we bump up to 21.0.7 now, as it seems to be the latest GA?